.changeset/tidy-numbers-rush.md

```diff
@@ -0,0 +1,7 @@
+---
+"@patternfly/pfe-tools": patch
+"@patternfly/pfe-core": patch
+"@patternfly/elements": patch
+---
+
+Update dependencies, including Lit version 3
```

Comment:
Didn't we decide in chat that this bump of lit 3 would become a `major`?

Reply:
This is my only blocker to approval.  I don't have a huge argument other then signaling a major change to the lit dependency as a big part of this project.  I can't live with whatever decision is made.  Either way the base class removal will be another major release that will follow shortly.

Reply:
I left an approval, so will just need to resolve this convo to merge.

Reply:
Here is the suggestion of the change requested for clarification for those following along.
```Suggestion
"@patternfly/elements": major
---

Update dependencies, including Lit version 3
```